modules/kmm-uninstalling-kmmo-red-hat-catalog.adoc

```diff
@@ -0,0 +1,15 @@
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-uninstalling-kmmo-red-hat-catalog_{context}"]
+= Uninstalling a Red Hat catalog installation
+
+Use this procedure if KMM was installed from the Red Hat catalog.
+
+.Procedure
+
+Use one of the following methods to uninstall the KMM Operator:
+
+* Use the OpenShift console under *Operators* --> *Installed Operators* (preferred).
+
+* Delete the `Subscription` resource in the KMM namespace.
```

Comment:
Technically, the first line is the step. This is a little format-heavy for such a small procedure, I know :bow: 
```suggestion
* Use one of the following methods to uninstall the KMM Operator:

** Use the OpenShift console under *Operators* --> *Installed Operators* (preferred).

** Delete the `Subscription` resource in the KMM namespace.
```

One possible alternative, which might be better since we typically want to document the preferred method primarily if not exclusively:
```suggestion
* Use the OpenShift console under *Operators* --> *Installed Operators*.
+
[NOTE]
====
Alternatively, you can delete the `Subscription` resource in the KMM namespace.
====
```

Whatever you decide here, is there a more specific description of what the user needs to do than "use the console"? I'm guessing there is some sort of "Uninstall" button or something.

Reply:
Yeah, I was also conscious of the  format-heaviness. I like your second suggestion about using the Note. I also changed this text to "Use the OpenShift console under *Operators* --> *Installed Operators* to locate and uninstall the Operator."